test/tests/test-helper.js

```diff
@@ -1,15 +1,12 @@
 import Schema from 'ember-orbit/schema';
 import Store from 'ember-orbit/store';
 
-var get = Ember.get,
-    set = Ember.set;
-
 var createStore = function(options) {
   options = options || {};
 
   var container = new Ember.Container();
```

Comment:
I'm hoping to clean up the registry / container mess in ember itself with https://github.com/emberjs/ember.js/pull/11440

Since that's not yet merged, I'll try to clean up any remaining deprecations for 1.13.